homeassistant/components/media_player/plex.py

```diff
@@ -145,6 +145,10 @@ def update_devices():
             _LOGGER.error("Could not connect to plex server at http://%s",
                           host)
             return
+        except requests.exceptions.RequestException as ex:
```

Comment:
I think that you should remove the catching of `OSError`, because that never happens.